Expensify/App
# Make Policy Rooms available in Search and LHN

@joelbettner, please review when you get the chance
CC: @jasperhuangg @yuwenmemon 

### Details
Adds the rest of the UI for policyRooms so that we can see it in the header. This PR does not include the Report Details that'll be in a separate PR.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/176851

### Tests


### QA Steps
Verify that now when a policy room is created, we can see it in the LHN and in search.
1. Make sure you have a workspace already, then click on + in bottom left and choose New Room
2. Name it something and select your workspace
![image](https://user-images.githubusercontent.com/19364431/148108761-23eb8e56-f4c1-491c-8a1c-02e8c8da77cf.png)
3. After it's created verify you see this:
![image](https://user-images.githubusercontent.com/19364431/148108828-fd8dee13-b1b0-45a6-b301-c8a840a1364f.png)
4. Verify the report details look like this:
![image](https://user-images.githubusercontent.com/19364431/148108960-e4a48ed4-564e-4b54-bcb1-c06a0fc03b60.png)
(Another PR is changing this so it might look different)
5. Send a message in the room and verify it works.

6. Verify that you can search for the room.

### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/19364431/146462032-f9d39cd3-c485-46de-8b6e-3adb71c78af1.png)
